test/cpp/end2end/hybrid_end2end_test.cc

```diff
@@ -207,6 +207,9 @@ class HybridEnd2endTest : public ::testing::Test {
     ServerBuilder builder;
     builder.AddListeningPort(server_address_.str(),
                              grpc::InsecureServerCredentials());
+    // Always add a sync unimplemented service: we rely on having at least one
+    // synchronous method to get a listening cq
+    builder.RegisterService(&unimplemented_service_);
```

Comment:
Do we really need this to get a listening cq ? - the hybrid server already has a "sync part" and that should give us a listening cq anyway.

On a related note: I thought that these tests might have been previously failing because non-listening completion queues were not being added to `server->cqs` and hence weren't getting picked up for queuing completed events. I see that you fixed that (i.e add non-listening cqs to `server-cqs` and don't start a listener in `grpc_server_start` function on those pollsets) 

So shouldn't that fix take care of the hybrid_end2end test failures ?